pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv
+, fetchFromGitHub
+, intltool
+, bash
+, which
+, parallel
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gnome3
+, gtk-engine-murrine
+, gdk-pixbuf
+, librsvg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "3.1.2";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "gtk-theme";
+    rev = "d485d9aa2cf732c691260b29385807ab99b48e03";
```

Comment:
You can use `rev = version` here to fetch the archive from upstream `3.1.2` tagged release.

Reply:
Fixed it !